cmd/ipi-deprovision/ipi-deprovision.sh

```diff
@@ -53,6 +53,24 @@ EOF
   echo "will deprovision GCE cluster ${infraID} in region ${region}"
 done
 
+libvirt_p_cluster_age_cutoff="$(TZ=":America/New_York" date --date="-11 hours" '+%Y-%m-%dT%H:%M%z')"
+echo "deprovisioning clusters with creation time before ${libvirt_p_cluster_age_cutoff} in libvirt-p"
+for network in $(ssh ${bastion_p} virsh net-list --all --name | grep -v default) ; do
+        if [[ ! -z $(ssh ${bastion_p} journalctl -u libvirtd.service --until \"11 hours ago\" 2>/dev/null | grep ${network}) ]] ; then
```

Comment:
You are trying to check if the log contains the network? You also want to actually make use of the time cutoff variable:


```sh
if ssh "${bastion_p}" -- journalctl --unit libvirtd.service --until "${libvirt_p_cluster_age_cutoff}" 2>&/dev/null | grep -q "${network}"; then
```

Reply:
yes, that's the only way i can think of to know how old something is (if it was created before a certain time). 

and i can double-check, but i think i tried using that, but iirc the timestamp format's different than what journalctl expects -- so i was just using that var as a logging output. `--until` is so much simpler, it's harder to mess up, so I just went with that. do you mind if i keep that strategy?

Reply:
```
journalctl -u libvirtd --until $libvirt_p_cluster_age_cutoff
Failed to parse timestamp: 2020-04-23T04:48-0400
```

Reply:
The format of `$libvirt_p_cluster_age_cutoff` is entirely up to you -- I hope we could use it (and format it correctly) -- it is very useful to see the value about to be used printed in the logs.

Reply:
I'm also way more worried about the Bash on this line than the format of the time -- the `[[ ! -z` is not following best practices, please try out my suggestion.

Reply:
re time cutoff format: so can i just set it to `4 hours ago` then, since that's what journalctl is using?
re bash: didn't catch that you were talking about two things there. i'll fix that, thanks.